src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User attemptingUser = userService.findByEmail(email);
```

Comment:
Generally it is not wrong, however from my perspective, it is more conventional to name this variable like `user` or `foundUser`.